### PR TITLE
Added KDE.KDEConnect version 21.08.1.714

### PR DESCRIPTION
**Pull request by ImJoakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27610)

### CODE REVIEW

File: manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KDE.KDEConnect
+PackageVersion: 21.08.1.714
```
Comment (KaranKad) [inline manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.installer.yaml:4]:
```suggestion
PackageVersion: 21.08.1
```
image

File: manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: KDE.KDEConnect
+PackageVersion: 21.08.1.714
```
Comment (KaranKad) [inline manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 21.08.1
```

File: manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: KDE.KDEConnect
+PackageVersion: 21.08.1.714
```
Comment (KaranKad) [inline manifests/k/KDE/KDEConnect/21.08.1.714/KDE.KDEConnect.yaml:4]:
```suggestion
PackageVersion: 21.08.1
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
Looks like this issue is similar to https://github.com/microsoft/winget-pkgs/issues/18448

**Comment by ImJoakim** [comment]
Simon Redman, [15.09.21 01:33]

> I doubt that it's possible to have that specific information in the build number... I don't know whether the CI gives that information "back" at build-time, which it would need to in order to be correctly put in the installer

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**